trino.yaml

```diff
@@ -32,49 +32,36 @@ pipeline:
       tag: ${{package.version}}
       expected-commit: 15024442d61e68bc1bb91fa472fc491dadd9d042
 
+  - uses: patch
+    with:
+      patches: CVE-2023-36478.patch CVE-2023-5072.patch
+
   - runs: |
-      export JAVA_HOME=/usr/lib/jvm/java-17-openjdk
-      export LANG=en_US.UTF-8
-      export INCLUDE="-I${JAVA_HOME}/include -I${JAVA_HOME}/include/linux"
-      export CFLAGS="-Wall -Werror -fPIC -shared ${INCLUDE}"
+      set -x
 
-      mkdir -p ${{targets.destdir}}/data/trino
-      mkdir -p ${{targets.destdir}}/etc/trino/catalog
-      mkdir -p ${{targets.destdir}}/usr/bin
-      mkdir -p ${{targets.destdir}}/usr/lib/trino/bin
-      mkdir -p ${{targets.destdir}}/usr/lib/trino/lib
-      mkdir -p ${{targets.destdir}}/usr/lib/trino/plugin
-      mkdir -p ${{targets.destdir}}/var/lib/trino
-      mkdir -p ${{targets.destdir}}/etc/security/
-      mkdir -p /home/build/trino-all
-      mkdir -p /home/build/plugins/default/
-      mkdir -p /home/build/plugins/optional/
+      export JAVA_HOME=/usr/lib/jvm/java-17-openjdk
 
-      cd /home/build/
-      mvn package -q -DskipTests -T $(nproc) -pl ':trino-server'
-      tar -C /home/build/trino-all/ --strip-components 1 -xf core/trino-server/target/trino-server-${{package.version}}.tar.gz
+      ./mvnw package -q -DskipTests -T 8  -pl '!:trino-docs,!:trino-server-rpm'
 
-      mvn package -q -DskipTests -T $(nproc) -pl ':trino-cli'
-      chmod +x client/trino-cli/target/trino-cli-${{package.version}}-executable.jar
+      mkdir -p ${{targets.destdir}}/usr/lib/trino
+      cp ./core/trino-server/target/trino-server-${{package.version}}/NOTICE ${{targets.destdir}}/usr/lib/trino/
+      cp ./core/trino-server/target/trino-server-${{package.version}}/README.txt ${{targets.destdir}}/usr/lib/trino/
+      cp -r ./core/trino-server/target/trino-server-${{package.version}}/bin ${{targets.destdir}}/usr/lib/trino/bin/
+      cp -r ./core/trino-server/target/trino-server-${{package.version}}/lib ${{targets.destdir}}/usr/lib/trino/lib/
 
-      rsync -av /home/build/trino-all/plugin/* /home/build/plugins/optional/
-      rm -rf /home/build/trino-all/plugin/*
-      rsync -av --ignore-existing /home/build/trino-all/* ${{targets.destdir}}/usr/lib/trino/
-      # rsync -av --ignore-existing /home/build/plugins/default/* ${{targets.destdir}}/usr/lib/trino/plugin/
+      # Copy default configurations.
+      mkdir -p ${{targets.destdir}}/etc
+      cp -r core/docker/default/etc ${{targets.destdir}}/etc/trino
 
-      rsync -av --ignore-existing core/docker/default/etc/* ${{targets.destdir}}/etc/trino/
-      rsync -av --force core/docker/bin/* ${{targets.destdir}}/usr/lib/trino/bin/
-      rsync -av --ignore-existing client/trino-cli/target/trino-cli-${{package.version}}-executable.jar ${{targets.destdir}}/usr/bin/
-      rsync -av --ignore-existing /usr/lib/libjvmkill.so ${{targets.destdir}}/usr/lib/trino/bin/
+      # Copy docker helper scripts.
```

Comment:
I think we'll want to split some of this out into an oci-entrypoint style package e.g. https://github.com/wolfi-dev/os/blob/15c516cac756936e157b3e05166734e7b20fbfca/neo4j.yaml#L49

And a config package: https://wiki.inky.wtf/docs/teams/engineering/images/package-and-image-guides/serviceable-packages/

Reply:
Done, great point!